ebpf.go

```diff
@@ -64,13 +66,21 @@ int enter_trace(struct tracepoint__raw_syscalls__sys_enter* args)
 {
     struct syscall_data data = {};
     u64 key = 0;
+    u32 seen = 0xFF;
     unsigned int zero = 0;
     struct task_struct *task;
     struct nsproxy *nsproxy;
     struct mnt_namespace *mnt_ns;
+    int id = (int)args->id;
+
+    if (seen_syscalls.lookup(&id) == seen) {
```

Comment:
lookup returne 0xFF on success?

Reply:
Ok any reason to select 0xFF rather then just 1?

Reply:
I've simplified it to not use `0xFF` but the syscall number value itself, as anyway the value is ignored